closure/testing/test/BUILD

```diff
@@ -111,6 +111,40 @@ closure_js_test(
     ],
 )
 
+# TEST: closure_js_library targets from exports should be provided when used in
+# a data attr.
+
+closure_js_library(
+    name = "external",
+    srcs = ["external.js"],
+)
+
+closure_js_library(
+    name = "all_external",
```

Comment:
Let's put a more describing name; point is exporting right?

Reply:
`exported_external`?

Reply:
exports_data?

Reply:
exports_data?

Reply:
updated